extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select All Elements Of Same Family.pushbutton/script.py

```diff
@@ -13,7 +13,10 @@
 
 for el in selection:
     try:
-        family = el.Symbol.Family
+        if isinstance(el, DB.FamilySymbol):
```

Comment:
Support Family Symbol as input. It allows user to select Family Type from project browser (the same below)